src/main/java/com/example/monitoring/core/api/config/SecurityConfiguration.java

```diff
@@ -23,9 +23,11 @@ public class SecurityConfiguration {
     public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Exception {
         http.cors(AbstractHttpConfigurer::disable)
                 .csrf(AbstractHttpConfigurer::disable)
-                .authorizeHttpRequests(auth -> auth
-                        .requestMatchers("/api/v1/**").permitAll()
-                        .anyRequest().authenticated())
+                .authorizeHttpRequests(auth -> {
+                        auth.requestMatchers("/api/v1/alert/**").permitAll(); //TODO: add authentication to alerts
+                        auth.requestMatchers("/api/v1/**").permitAll()
+                        .anyRequest().authenticated();
+                        })
```

Comment:
Please remove.
Path /api/v1/** is okay for now. It is changing in the upcoming auth0-impl PR so less conflicts to resolve when no filter chain (SecurityConfiguration) is affected.